### ISSUE TITLE
Crash with The Graveyard mod 

### ISSUE THREAD

**Issue by finallion** [Globox1997/RpgDifficulty#24]
Minecraft 1.19
The Graveyard 1.10
RpgDifficulty 1.1.0

Hey, some people are reporting a crash happening when RPGDifficulty and The Graveyard are present. 
The crash happens inside a graveyard entity at the initialize method. But the referenced method does nothing more than calling the vanilla super method, so I assume there is something going wrong with the Mixins to MobEntity.

[crash-2022-06-18_17.57.04-server.txt](https://github.com/Globox1997/RpgDifficulty/files/8933926/crash-2022-06-18_17.57.04-server.txt)

[latest.log](https://github.com/Globox1997/RpgDifficulty/files/8933927/latest.log)

Link to the Graveyard mob class: https://github.com/finallion/The_Graveyard_-FABRIC-/blob/1.19/src/main/java/com/finallion/graveyard/entities/HordeGraveyardEntity.java

**Comment by Globox1997** [comment]
Thanks for reporting, fixed with [https://github.com/Globox1997/RpgDifficulty/commit/638412d4d6efeb1a2423570fcc0e8e74887fe478](https://github.com/Globox1997/RpgDifficulty/commit/638412d4d6efeb1a2423570fcc0e8e74887fe478)

**Comment by Wilevar** [comment]
Still seems to be an issue in the build released an hour ago.
I have a server world with many mods, including The Graveyard 1.10, and adding RPGDifficulty to it makes the server crash as soon as a player attempts to load. Possibly because Graveyard mobs exist in the chunks being loaded. I'll attach the crash report:

[crash-2022-06-19_10.23.25-server.txt](https://github.com/Globox1997/RpgDifficulty/files/8935721/crash-2022-06-19_10.23.25-server.txt)

I removed Graveyard to test, it works with RPG Difficulty.
It worked before adding RPG Difficulty, and resumes working without it. I added no other mods in this test.

Let me know if I can send any other info to help narrow down the issue.
